tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -60,19 +54,29 @@ using gpuStream_t = hipStream_t;
 using gpuError_t = hipError_t;
 #endif
 
-#define GetGPUStream(context) context->eigen_gpu_device().stream()
+#if GOOGLE_CUDA
+// cudaGetErrorString is available to both host and device
```

Comment:
This change seems unrelated. Is it inherited from a different PR that is supposed to go in first?

Reply:
<backgroud> I don't know why it ended up in this location in our develop-upstream branch, because the definition started in the gpu_launch_config.h. It might be handled differently in our weekly merge task.

I'd assume you prefer it in non-macro format as well. Will update in next commit.

Reply:
#28568 commit a7137e1 addressed the review by moving GetGpuStream() from gpu_launch_config.h to gpu_kernel_helper.h

Will remove the wrong `GetGPUStream()` macro.

Reply:
Let's just leave it for now.